libcxx/include/__math/traits.h

```diff
@@ -45,12 +45,7 @@ _LIBCPP_NODISCARD inline _LIBCPP_HIDE_FROM_ABI bool signbit(_A1) _NOEXCEPT {
 
 // isfinite
```

Comment:
Is the `<limits>` include still required?

Reply:
Good catch! No, it's not needed anymore. Corrected.